docs/developer/prebuilt_world_loading.md

```diff
@@ -1,94 +1,126 @@
-Loading pre-built Worlds.
-=========================
+<h1>  Loading Pre-Built Worlds </h1>
```

Comment:
While the `<h1>` tag does achieve the formatting desired, the website building program, [Sphinx](http://www.sphinx-doc.org/en/master/) needs to have the header in either Markdown or ReStructuredText format to be able to correctly parse the file and automatically add it to the table of contents. So please replace this line with either

```markdown
Loading Pre-Built Worlds
========================
```

or 

```markdown
# Loading Pre-Built Worlds
```

Here's a [markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for quick reference.